sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1897,6 +1897,14 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  // SPARK-38809 - Config option to allow skipping null values for hash based stream-stream joins.
```

Comment:
nit.  Can you briefly describe in what situations null can occur in stream-stream joins?

Reply:
Done